.github/workflows/gpu-tests.yml

```diff
@@ -121,7 +121,7 @@ jobs:
 
       - name: Run GPU Unit Tests
         continue-on-error: false
-        uses: nick-fields/retry@v3
+        uses: nick-fields/retry@14672906e672a08bd6eeb15720e9ed3ce869cdd4
```

Comment:
Can we specify a version like `v2` or v3 with minor and patch instead of a commit ?

Reply:
yep, looks like 2.9.0 is a sufficient downgrade.